src/app/joint-control/joint-control-core/motor-settings/motor-settings.component.ts

```diff
@@ -38,6 +38,18 @@ export class MotorSettingsComponent {
     ngOnInit(): void {
         this.motorService
             .getSettingsObservable(this.motor.sourceMotorName)
+            .pipe(
+                map((settings) => {
+                    settings = structuredClone(settings);
+                    settings.rotationRangeMin = Math.floor(
+                        settings.rotationRangeMin / 100,
```

Comment:
the same object is published to every other subscriber, so other subscriber's settings would also be altered. I don't think it currently makes a difference, but I think to prevent possible future bugs, we should perform the deep-cloning step.